nailgun/entities.py

```diff
@@ -1452,7 +1461,8 @@ def create_payload(self):
         return payload
 
 
-class HostGroupClasses(Entity):
+class HostGroupClasses(
+        Entity, EntityCreateMixin, EntityDeleteMixin, EntityReadMixin):
     """A representation of a Host Group Classes entity."""
 
     def __init__(self, server_config=None, **kwargs):
```

Comment:
I see a reference to `self.hostgroup` below. Make a call to `_check_for_value` here, like so:

```python
def __init__(self, server_config=None, **kwargs):
    _check_for_value('hostgroup', kwargs)
    …
```

Doing this ensures that `self.hostgroup` will be available for use when populating `self._meta['api_path']`.